src/2d/spread2d_wrapper.cu

```diff
@@ -65,6 +65,15 @@ int cufinufft_spread2d(int ms, int mt, int nf1, int nf2, CPX* h_fw, int M,
 		2*M*sizeof(FLT)+M*sizeof(CUCPX), milliseconds);
 #endif
 
+	if(d_plan->opts.gpu_method == 1){
```

Comment:
So this is not related to CUB/Thrust, right? It's fine to include it in this PR, but would be good to separate these things in the future.

Reply:
You're right. That change is not actually related to this pull request. Ok, I will do that in the future.